### PR TITLE
Webhook event types

### LINKED ISSUE #440: Missing type definitions for webhook events
I'm trying to find type definitions for contents of the webhook events, but can't seem to find them.

Please point me in the right direction, or add them if they do not exist.

Thanks!

### PR DESCRIPTION
**Pull request by bodrovis**
Closes #440

### CODE REVIEW

File: docs/additional_info/changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 13.0.0
```
Comment (kibertoad) [inline docs/additional_info/changelog.md:3]: why semver major?

Comment (bodrovis) [reply docs/additional_info/changelog.md:3]: I'm actually now entirely sure. This is a new feature, not anything breaking, so theoretically we could mark it as 12.x. Still, this is a relatively "big" thing. In any case, this pr is not finalized, I'll need support from someone who could check if any fields are missing from these schemas

Comment (kibertoad) [reply docs/additional_info/changelog.md:3]: yeah, I's recomment semver minor for non-breaking incremental additions

Comment (bodrovis) [reply docs/additional_info/changelog.md:3]: Okay

**Pull request merged**